docs/user_guides/fs/feature_view/batch-data.md

```diff
@@ -16,6 +16,26 @@ It is very common that ML models are deployed in a "batch" setting where ML pipe
     Dataset<Row> ds = featureView.getBatchData("20220620", "20220627")
     ```
 
+## Retrieve batch data with primary keys and event time
+In certain scenarios, for example for time series model input data needs to be sorted according to primary key(s) and event time combination.  
+Or one might want to merge predictions back to original input data for postmortem analysis. However,  primary key(s) and event time usually are not included in the feature view query as 
+they are not features used for training. To get them pass following attributes to `get_batch_data` method:  
+`primary_keys=True` and/or `event_time=True`.
+
+=== "Python"
+    ```python
+    # get batch data
+    df = feature_view.get_batch_data(
+    start_time = "20220620",
+    end_time = "20220627",
+    primary_keys=True,
+    event_time=True
+    ) # return a dataframe with primary keys and event time
+    ```
+!!! note
+    If event time column has the same name in feature groups included in feature view query then the event time of the left most feature group in the query will be returned. If they have different names then
+    all of them will be returned. Join prefix doesn't have any influence on this behaviour. 
```

Comment:
```suggestion
    If the event time columns have the same name across all the feature groups included in the feature view, then only the event time of the label feature group (left most feature group in the query) will be returned. If they have different names, then all of them will be returned. The Join prefix does not have any influence on this behaviour.
```

Tbh I struggle to find the rationale for this behaviour. Why would it matter if the names of the event times are the same or different? Even if the column names are the same, the times might be different, so we either show all of them or only the label feature group. But I don't see the correlation between the names and deciding which column to show.